notebooks/Datasources.ipynb

```diff
@@ -0,0 +1,393 @@
+   "metadata": {},
+   "source": [
+    "### NeomarilDataSource\n",
+    "\n",
+    "It's where you can register your datasource."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:19:43.230 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:19:43.238 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n",
+      "2024-03-20 19:19:44.777 | INFO     | neomaril_codex.datasources:register_datasource:101 - DataSource 'testeDataSouce' was registered!\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<neomaril_codex.datasources.NeomarilDataSource at 0x7f6ba047e170>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "credentials_path = os.path.abspath('./samples/datasources/credentials.json')\n",
+    "\n",
+    "client.register_datasource(\n",
+    "    datasource_name='testeDataSouce',\n",
+    "    provider='GCP',\n",
+    "    cloud_credentials=credentials_path,\n",
+    "    group='datarisk'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List Datasources\n",
+    "\n",
+    "Use this function to find your datasouce."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'Name': 'testeDataSouce',\n",
+       "  'Group': 'datarisk',\n",
+       "  'Provider': 'GCP',\n",
+       "  'RegisteredAt': '2024-03-20T22:19:44.745936+00:00'}]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.list_datasource(provider='GCP', group='datarisk')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get Datasources\n",
+    "\n",
+    "Use this function to get your datasouce neomaril object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:19:50.600 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:19:50.604 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n"
+     ]
+    }
+   ],
+   "source": [
+    "datasource = client.get_datasource(datasource_name='testeDataSouce', provider='GCP', group='datarisk')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### NeomarilDataset\n",
+    "\n",
+    "It's where you can import your dataset."
```

Comment:
I would make clear at the message that you import the dataset to be inside the data source